build-with-pip3.file

```diff
@@ -31,7 +39,7 @@ Requires: python3 py3-setuptools
 BuildRequires: py3-pip
 
 %if "%{?source0:set}" != "set"
-%define source0 pip://%{pip_name}/%{realversion}?pip_options=%{PipDownloadOptions}&pip=pip3&output=/%{pkgsource}
+%define source0 pip://%{pip_name}/%{realversion}?pip_options=%{PipDownloadOptions}&pip=pip3&package_dependency=py3-pip&output=/%{pkgsource}
```

Comment:
this change here looks good and this should instruct cmsBuild to use py3-pip env for downloading.

Reply:
Sigh... sorry for the noise. I did not have this change in the branch I'm currently testing! You're the master Shahzad! Thanks